data_collection/gazette/spiders/to/to_parana.py

```diff
@@ -0,0 +1,14 @@
+import datetime as dt
+
+from gazette.spiders.base.nucleogov import NucleoGovGazetteSpider
+
+
+class ToParanaSpider(NucleoGovGazetteSpider):
+    name = "to_parana"
+    TERRITORY_ID = "1716208"
+    allowed_domains = [
+        "diariooficial.parana.to.gov.br",
+    ]
+    url_base = "https://diariooficial.parana.to.gov.br/api/diarios?data={}&calendar=true&situacao=2"
+
```

Comment:
Reflete aqui a mudança na classe base

```suggestion
    url_base = "https://diariooficial.parana.to.gov.br"
```